### PR TITLE
Added export erase data settings in admin settings

### PR DESCRIPTION
**Pull request by malavasita**
Under Export/Import settings tab user will have an additional option of Export/Erase personal data which will redirect to Wordpress export and erase data tab respecively
Task: #1109

### CODE REVIEW

File: app/assets/admin/js/settings.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ rtMediaAdmin.templates = {
 };
 
 jQuery( document ).ready( function ( $ ) {
-	
+
 	if ( $( '#rtm-licenses' ).length ) {
 		$( '#rtm-licenses .license-inner:first input:first' ).focus();
```
Comment (SID177) [inline app/assets/admin/js/settings.js]: Take one variable i.e. `var a = $( '#rtm-licenses' )` and then use its other properties
Use `a.find()` for its children.
@malavasita

File: app/assets/admin/js/settings.js [hunk 2]
```diff
@@ -961,6 +962,15 @@ jQuery( document ).ready( function ( $ ) {
 			downloadAnchorNode.remove();
 		} );
 	} );
+
+	jQuery( '#rtm-export-data-button' ).click( function(){
```
Comment (chandrapatel) [inline app/assets/admin/js/settings.js]: Isn' this anchor tag?

Comment (malavasita) [reply app/assets/admin/js/settings.js]: Yes. But we wanted to redirection to `Export Personal Data` and `Erase Personal Data` page on click. SO we have made it Button link.
@chandrapatel

File: app/admin/RTMediaFormHandler.php [hunk 3]
```diff
@@ -1511,3 +1536,4 @@ class="rtm-field-wrap"><?php call_user_func( $option['callback'], $option['args'
 		}
 	}
 }
+new RTMediaFormHandler();
```
[the remark this answers is not in the document]
Comment (malavasita) [reply app/admin/RTMediaFormHandler.php]: Added new line at end @milindmore22 :+1:

**Pull request merged**